exopy/app/headless/manifest.enaml

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2021 by Exopy Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""Headless plugin manifest.
+
+"""
+import time
+import os
+import json
+
+from enaml.workbench.api import PluginManifest, Extension
+from enaml.workbench.ui.api import ActionItem
+from enaml.workbench.core.api import Command
+
+from ...measurement.measurement import Measurement
+from ..api import AppStartup
+from ...measurement.hooks.api import PostExecutionHook
+
+from ..headless.autoclose import AutoClose
+
+PLUGIN_ID ='exopy.app.headless'
+
+
+# =============================================================================
+# --- Factories ---------------------------------------------------------------
+# =============================================================================
+
+def log_plugin_factory():
+    """ Factory function for the LogPlugin.
+
+    """
+    from .plugin import LogPlugin
+    return LogPlugin()
```

Comment:
What is the point ? If your plugin is completely empty I believe you can just skip the factory.